packages/swarm-components/src/LinkButton.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+import * as React from 'react';
+import Icon from './Icon';
+import {
+	FILLS,
+	getButtonType,
+	getSwarmSize,
+	getIconPosition,
+} from './utils/buttonUtils';
+
+export type LinkButtonProps = {
+	/**
+	 * The bordered button
+	 */
+	bordered?: boolean,
+
+	/**
+	 * Classes for additional styles to be applied
+	 */
+	className?: string,
+
+	/**
+	 * Indicates whether the button is disabled or not
+	 */
+	disabled?: boolean,
+
+	/**
+	 * Supports icon classes found at https://meetup.github.io/swarm-icons/
+	 */
+	icon?: string,
+
+	/**
+	 * Inverted styles for dark backgrounds
+	 */
+	inverted?: boolean,
+
+	/**
+	 * The neutral style
+	 */
+	neutral?: boolean,
+
+	/**
+	 * The function invoked when interacting with Button
+	 */
+	onClick?: () => {},
+
+	/**
+	 * The primary style
+	 */
+	primary?: boolean,
+
+	/**
+	 * Aligns the icon to the right
+	 */
+	right?: boolean,
+
+	/**
+	 * Use the small button size
+	 */
+	small?: boolean,
+
+	/**
+	 * Content of button
+	 */
+	children: React.Node,
+};
+
+const LinkButton = (props: LinkButtonProps): React.Element<'a'> => {
+	// destructuring to not pass invalid attributes to node
+	const {
+		icon,
+		right,
+		children,
+		...other
+	} = props;
+
+	const buttonType = getButtonType(props);
+
+	return (
+		<a
+			data-swarm-button={buttonType}
```

Comment:
could we use `data-swarm-link` here? `@extend` in postcss should help. https://github.com/jonathantneal/postcss-extend-rule